File: csharp/ButtplugCSharpFFI/ButtplugMessageException.cs
using System;

namespace Buttplug
{
    public class ButtplugMessageException : ButtplugException
    {
        /// <inheritdoc />
        public ButtplugMessageException()
        {
        }

        /// <inheritdoc />
        public ButtplugMessageException(string aMessage) : base(aMessage)
        {
        }

        /// <inheritdoc />
        public ButtplugMessageException(string aMessage, Exception aInner) : base(aMessage, aInner)
        {
        }
    }
}

File: java/lib/src/main/java/io/buttplug/exceptions/ButtplugMessageException.java
package io.buttplug.exceptions;

public class ButtplugMessageException extends ButtplugException {
    public ButtplugMessageException(String msg) {
        super(msg);
    }

    public ButtplugMessageException(String msg, Exception inner) {
        super(msg, inner);
    }
}
